packages/typescript-estree/tests/ast-alignment/fixtures-to-test.ts

```diff
@@ -364,46 +356,32 @@ tester.addFixturePatternConfig('typescript/basics', {
     'abstract-class-with-optional-method', // babel parse errors
     'declare-class-with-optional-method', // babel parse errors
     /**
-     * PR for parameter property ranges has been merged into Babel: https://github.com/babel/babel/pull/9284
-     * TODO: remove me in next babel > 7.2.3
+     * Was expected to be fixed by PR into Babel: https://github.com/babel/babel/pull/9284
+     * But not fixed in Babel 7.3
+     * TODO: Investigate differences
      */
     'class-with-private-parameter-properties',
     'class-with-protected-parameter-properties',
     'class-with-public-parameter-properties',
     'class-with-readonly-parameter-properties',
     /**
-     * PR for type import has been merged into Babel: https://github.com/babel/babel/pull/9302
-     * TODO: remove me in next babel > 7.2.3
+     * Was expected to be fixed by PR into Babel: https://github.com/babel/babel/pull/9302
+     * But not fixed in Babel 7.3
+     * TODO: Investigate differences
```

Comment:
in babel i implemented this in a little diffrent way.

we have one node TSImportType with property isTypeOf set to true/false (as is in ts)
in babel its 2 nodes: typeof -> TSImportType